packages/components/src/components/Forms/FormText.tsx

```diff
@@ -0,0 +1,167 @@
+import { Text } from 'react-native'
+import React, { FC } from 'react'
+
+import { StringOrTextWithA11y } from '../../types/common'
+import { getA11yText, getDisplayText } from '../../utils'
+import { useTheme } from '../../utils'
+import { useTranslation } from 'react-i18next'
+
+export type FormTextProps = {
```

Comment:
Seems pretty purpose made, potentially:
```suggestion
type formTextProps = {
```

Reply:
Removed `export` to make it a local type